py3status/modules/nvidia_temp.py

```diff
@@ -43,33 +59,21 @@ def nvidia_temp(self):
         if temps != []:
             data = []
             for temp in temps:
-                fmt_str = "{temp}{format_units}".format(
-                    temp=temp,
-                    format_units=self.format_units
-                )
-                data.append(fmt_str)
-
-            output = "{format_prefix}{data}".format(
-                format_prefix=self.format_prefix,
-                data=self.temp_separator.join(data)
-            )
+                data.append(self.py3.safe_format(self.format_gpu_temp, {'temp': temp}))
 
+            output = "{data}".format(data=self.icon_separator.join(data))
```

Comment:
Actually this should work better
```
for temp in temps:
    data.append(self.py3.safe_format(self.format_temp, {'temp': temp}))

data = self.py3.composite_join(self.icon_separator, data)
```

`temp_data` may be a better name than `data`

Sorry I can't test this module